### PR TITLE
sci-libs/udunits: Bump to 2.2.28

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

File: sci-libs/udunits/udunits-2.2.28.ebuild [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+inherit fortran-2
+
+DESCRIPTION="Library for manipulating units of physical quantities"
+HOMEPAGE="https://www.unidata.ucar.edu/software/udunits/"
+SRC_URI="ftp://ftp.unidata.ucar.edu/pub/udunits/${P}.tar.gz"
+
+SLOT="0"
+LICENSE="UCAR-BSD"
+KEYWORDS="~alpha ~amd64 ~hppa ~mips ~ppc ~sparc ~x86 ~amd64-linux ~x86-linux"
```
Comment (SoapGentoo) [inline sci-libs/udunits/udunits-2.2.28.ebuild:14]: order

File: sci-libs/udunits/udunits-2.2.28.ebuild [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+src_configure() {
+	econf --disable-static
+}
+
+src_install() {
+	default
+
+	# no static archives
+	find "${D}" -name '*.la' -delete || die
```
Comment (SoapGentoo) [inline sci-libs/udunits/udunits-2.2.28.ebuild:27]: ED?

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: sci-libs/udunits/udunits-2.2.28.ebuild (hunk 2 above)
Comment (jsmolic) [reply sci-libs/udunits/udunits-2.2.28.ebuild]: Right, good old ED

File: sci-libs/udunits/udunits-2.2.28.ebuild (hunk 1 above)
Comment (jsmolic) [reply sci-libs/udunits/udunits-2.2.28.ebuild:14]: Fixed

**Review by SoapGentoo — APPROVED** [review]

**Pull request closed without merging**